src/fabric/src/fabric2_sup.erl

```diff
@@ -30,7 +30,7 @@ start_link(Args) ->
 
 init([]) ->
     config:enable_feature(fdb),
-    Flags = {one_for_one, 1, 5},
+    Flags = {one_for_all, 1, 5},
```

Comment:
I think we could probably use `rest_for_one` here. The idea there is that if any children at the top die, we'd restart all the ones that follow it, but not the ones before it. So if `fabric2_server` crashes, we'd restart everything else. But if `fabric2_db_expiration` crashes, we'd just restart it on its own.